src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,83 @@
+package mate.jdbc.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(Long id, String model, Manufacturer manufacturer, List<Driver> drivers) {
```

Comment:
Consider removing this constructor. It's never used and the 'id' parameter from the constructor is generated by DB and applied with setter.